From be3d217e813682e0be247ee03211e3abb13eb710 Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Fri, 2 Mar 2018 21:55:37 +1100
Subject: [PATCH] TRITON-198 - cloudapi test failures checking error message
 Reviewed by: Dave Eddy <dave.eddy@joyent.com> Approved by: Dave Eddy
 <dave.eddy@joyent.com>

---
 test/machines.test.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/machines.test.js b/test/machines.test.js
index 141cc58..e7b31c8 100644
--- a/test/machines.test.js
+++ b/test/machines.test.js
@@ -1088,7 +1088,7 @@ function (t) {
         t.ok(err, 'delete error expected');
         t.equal(res.statusCode, 409, 'http code');
         t.deepEqual(body, {
-            code: 'MachineUndeletableError',
+            code: 'CannotDestroyMachineError',
             message: 'Instance has "deletion_protection" enabled, ' +
                 'preventing deletion'
         }, 'check error message');
-- 
2.21.0

